src/locale/messages.fr.xlf

```diff
@@ -1,313 +1,365 @@
       </trans-unit>
       <trans-unit datatype="html" id="7104935342812835081">
         <source>Please set the api endpoint</source>
-        <target state="new">Merci de contacter l&apos;administrateur, un point d&apos;acces est nécessaire</target>
+        <target state="translated">Merci de contacter l'administrateur, un point d'accès est nécessaire</target>
       </trans-unit>
       <trans-unit datatype="html" id="994363f08f9fbfa3b3994ff7b35c6904fdff18d8">
         <source>Profile</source>
-        <target state="new">Profil</target>
+        <target state="translated">Profil</target>
       </trans-unit>
       <trans-unit datatype="html" id="918f28cae009f8e67af8ca86f26b47ed2ce0ae5e">
         <source>Program versions</source>
-        <target state="new">Versions du programme</target>
+        <target state="translated">Versions du programme</target>
       </trans-unit>
       <trans-unit datatype="html" id="06c663bf1474713f57551123a46b34318543b67d">
         <source>Reset</source>
-        <target state="new">Annuler</target>
+        <target state="translated">Réinitialiser</target>
       </trans-unit>
       <trans-unit datatype="html" id="a17349ecebaf5eb3bae3106c7c99f7978b9cb699">
         <source>Results from previous tests are not available</source>
-        <target state="new">Les résultats des tests précédents ne sont pas disponibles</target>
+        <target state="translated">Les résultats des tests précédents ne sont pas disponibles</target>
       </trans-unit>
       <trans-unit datatype="html" id="baa5cda119114bb5fba61496780778a5fa10caf1">
         <source>Run test</source>
-        <target state="new">Lancer un test</target>
+        <target state="translated">Lancer un test</target>
       </trans-unit>
       <trans-unit datatype="html" id="0bd8b27f60a1f098a53e06328426d818e3508ff9">
         <source>Share</source>
-        <target state="new">Partager</target>
+        <target state="translated">Partager</target>
       </trans-unit>
       <trans-unit datatype="html" id="a119d6c78f3f291d3be8bac176f93afe0757f615">
         <source>Sorry, there is nothing here!</source>
-        <target state="new">Désolé, il n&apos;y a rien ici!</target>
+        <target state="translated">Désolé, il n'y a rien ici !</target>
       </trans-unit>
       <trans-unit datatype="html" id="2e52adeb40004b84f94197ec742951783fb4e377">
         <source>Display all results</source>
-        <target state="new">Afficher l&apos;ensemble des résultats</target>
+        <target state="translated">Afficher l'ensemble des résultats</target>
         <note priority="1" from="description">tooltip all</note>
       </trans-unit>
       <trans-unit datatype="html" id="688e4555813c90de96d11699aef24aab49d5b4af">
         <source>Critical - means a very serious error.</source>
-        <target state="new">Critical - une erreur vraiment sévère</target>
+        <target state="translated">Critique - une erreur vraiment sévère.</target>
```

Comment:
Remove trailing dot

Reply:
I disagree; the original sentence has a trailing full stop, so the translation probably should too.

Maybe also an opportunity to make the translation more faithful to the original. How about: “Critique – une erreur très grave.”